### PR TITLE
Feature/42/create footer

### PR DESCRIPTION
**Pull request by PaulRomanov**
image

### CODE REVIEW

File: src/app/components/footer/footer.component.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+})
+export class FooterComponent implements OnInit {
+  ngOnInit(): void {
```
Comment (ozhurova) [inline src/app/components/footer/footer.component.ts:9]: Why do you need implements OnInit on footerComponent?

File: src/app/components/footer/footer.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    throw new Error('Method not implemented.');
+  }
+  // constructor() {}
```
Comment (ozhurova) [inline src/app/components/footer/footer.component.ts:12]: Please, clean up comments

Comment (PaulRomanov) [reply src/app/components/footer/footer.component.ts]: image

File: src/app/components/footer/footer.component.ts (hunk 1 above)
Comment (PaulRomanov) [reply src/app/components/footer/footer.component.ts]: fix code

**Review by mhasilau — APPROVED** [review]

File: src/app/app.module.ts [hunk 3]
```diff
@@ -8,18 +8,17 @@ import { ToastrModule } from 'ngx-toastr';
...
-  ],
+  declarations: [AppComponent, FooterComponent],
   imports: [
     CommonModule,
     BrowserAnimationsModule, // required animations module
     ToastrModule.forRoot(), // ToastrModule added
     BrowserModule,
     AppRoutingModule,
-    HeaderModule
+    HeaderModule,
```
Comment (vstrabko) [inline src/app/app.module.ts:21]: shouldn't we include footer in imports?

**Review by ozhurova — APPROVED** [review]

**Pull request merged**